spec/ADAS/ADAS.vspec

```diff
@@ -131,6 +143,18 @@ ObstacleDetection.IsError:
   type: sensor
   description: Indicates if obstacle sensor system incurred an error condition. True = Error. False = No Error.
 
+ObstacleDetection.ForwardDistance:
+  datatype: float
+  type: sensor
+  unit: m
+  description: Distance in meters to forward detected object
```

Comment:
- Is the idea that if `ObstacleDetection.IsWarning == False` then the value of this signal is not relevant, or do we need some special values (for example negative values) to represent that there is no obstacle present.
- How is this signal supposed to behave when reversing? - reporting obstacles behind vehicle or not reporting obstacles at all?
- Is it deliberately not specified how you calculate distances? Like if you are planning a turn (or the road turns), should it report the straight distance to the obstacle, or the distances you need to drive to hit the obstacle

Reply:
In the initial comment made with this PR and this one, very interested in how we can accommodate multitude of direction sensors on existing advanced (SAE AV 4+) or experimental vehicles. As the configurations will vary we cannot use simple directions (front, left, left front, etc) but instead could use degrees with 0 being directly in front of the vehicle. This compares to degrees on a compass (Vehicle.CurrentLocation.Heading in VSS). Sensor type would be of interest as we are seeing a mix.

Reply:
I assume the distance to object in front would relate to objects in the trajectory of the moving vehicle. So if there is a curve, stationary objects would not be relevant, if the trajectory is not hitting the object...